projects/bootloader/scripts/can_datagram.py

```diff
@@ -0,0 +1,343 @@
+# pylint: skip-file
+"""This client script handles datagram protocol communication between devices on the CAN."""
+
+import zlib
+import can
+
+DEFAULT_CHANNEL = "can0"
+PROT_VER = 1
+CAN_BITRATE = 500000
+
+MESSAGE_SIZE = 8
+HEADER_SIZE = 6
+
+
+class Datagram:
+    """This class acts as an easy modular interface for a datagram."""
+
+    def __init__(self, **kwargs):
```

Comment:
I mainly wrote this in because of the list of errors I was working through to get pylint to work. However, now that I've added: `# pylint: skip-file`, I no longer need to worry about the errors. Would you rather I list out those parameters, ignoring pylint?

Reply:
Eh, it's probably fine for now - listing them out would probably be the best practice but don't sweat about it